### PR TITLE
Fix broken compilation in v1.1.0

### PR DESCRIPTION
**Pull request by nacho-carnicero**
# Overview
Following the issue in #43 I am adding this PR in order to fix the compilation. Basically I:
 * Added missing files to the XCode project (the `RNCSegmentedControlManager.*` files were not added)
 * Changed the `React` imports in the project from `#import "RCTLibrary.h"` to `#import <React/RCTLibrary.h>`
 * Added a line in the documentation to let users know that they should link this library to their project to make it work. 
**NOTE:** I only added a link to the manually linking instructions, but this should be done in a better way (for instance by doing a `pod install` or something like this). I don't know very  much how installed libraries can be added to the Podfile of the root project when being installed so I did not add that change here, but this would be a nice enhancement to this PR.

# Test Plan

Basically I just tested that by making those changes my app which would not compile before now compiles.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ or npm:
 npm install --save @react-native-community/segmented-control
 ```
 
+Then you will need to link it to your project. You can follow the [manual linking](https://facebook.github.io/react-native/docs/linking-libraries-ios#manual-linking) instructions for that.
```
Comment (Naturalclar) [inline README.md]: This PR reminded me that there were no link instructions. :+1:
I'm going to change this part a bit.

```suggestion
## Link

- React Native 0.60+

run `cd ios && pod install`

- React Native <= 0.59

run `react-native link @react-native-community/segmented-control`

or you can follow the instructions to [manually link this package](https://facebook.github.io/react-native/docs/linking-libraries-ios#manual-linking).

## Upgrading to React Native _0.60+_

New React Native comes with `autolinking` feature, which automatically links Native Modules in your project. In order to get it to work, make sure you unlink `Segmented Control` first:

`react-native unlink @react-native-community/segmented-control`
```

File: ios/RNCSegmentedControl.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -23,9 +24,11 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
-		134814201AA4EA6300B7C361 /* libRNCSegmentedControl.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; includeInIndex = 0; path = libRNCSegmentedControl.a; sourceTree = BUILT_PRODUCTS_DIR; };
+		134814201AA4EA6300B7C361 /* libRNCSegmentedControl.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; includeInIndex = 0; name = libRNCSegmentedControl.a; path = libRNCViewpager.a; sourceTree = BUILT_PRODUCTS_DIR; };
```
Comment (Naturalclar) [inline ios/RNCSegmentedControl.xcodeproj/project.pbxproj]: this part seems to be adding RNCViewpager which is unrelated to this repository.

Comment (nacho-carnicero) [reply ios/RNCSegmentedControl.xcodeproj/project.pbxproj]: Indeed, for some reason the product name was set to RNCViewpager 🤷‍♂ 
I changed in with 76656e9

**Comment by Naturalclar** [comment]
Released v1.1.1 :tada:
